src/helix-ui/elements/index.js

```diff
@@ -4,6 +4,7 @@ export { HXAlertElement } from './HXAlertElement';
 export { HXBusyElement } from './HXBusyElement';
 export { HXCheckboxElement } from './HXCheckboxElement';
 export { HXDisclosureElement } from './HXDisclosureElement';
+export { HXDivElement } from './HXDivElement';
```

Comment:
This might have accidentally gotten dropped as a result of a rebase involving #309